nbconvert/exporters/pdf.py

```diff
@@ -175,7 +175,7 @@ def from_notebook_node(self, nb, resources=None, **kw):
                 rc = self.run_latex(tex_file)
             
             pdf_file = notebook_name + '.pdf'
-            if not os.path.isfile(pdf_file):
+            if not rc or not os.path.isfile(pdf_file):
```

Comment:
It would be clearer to write `rc != 0` since it's an integer for sure.

Reply:
Won't help with the test failures though.

Reply:
Looks like run_command does return a boolean,
https://github.com/jupyter/nbconvert/blob/master/nbconvert/exporters/pdf.py#L85